<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Dialog</class>
 <widget class="QDialog" name="Edit_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit_editcode"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_edititem"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_editminstock"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>code</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>item</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>minimum stock</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_editmrp"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>mrp</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_editexpiry"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>expiry in months</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_edit</sender>
   <signal>accepted()</signal>
   <receiver>Edit_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_edit</sender>
   <signal>rejected()</signal>
   <receiver>Edit_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
